### PR TITLE
Task/759

### PR DESCRIPTION
**Pull request by junminahn**
- update minor ui texts on request edit page
- allow delete the first redirect uri if has more than two items

### CODE REVIEW

File: app/form-components/ArrayFieldTemplate.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
+import React from 'react';
```
Comment (thegentlemanphysicist) [inline app/form-components/ArrayFieldTemplate.tsx:1]: Good code clean up

File: app/next.config.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ module.exports = {
     sso_authorization_response_type: process.env.SSO_AUTHORIZATION_RESPONSE_TYPE || 'code',
     sso_authorization_scope: process.env.SSO_AUTHORIZATION_SCOPE || 'openid',
     sso_token_grant_type: process.env.SSO_TOKEN_GRANT_TYPE || 'authorization_code',
+    kc_idp_hint: process.env.KC_IDP_HINT || '',
```
Comment (thegentlemanphysicist) [inline app/next.config.js:18]: Do we need to update our env files for local development?

**Review by thegentlemanphysicist — APPROVED** [review]
Looks good, though do we need to change our local env files at all?

File: app/next.config.js (hunk 2 above)
Comment (junminahn) [reply app/next.config.js:18]: yes, in your `.env.local`, add the environment variable
```
KC_IDP_HINT=idir
```

**Pull request merged**